test-integration/repay-loan/2_repay_loan_4_payments.js

```diff
@@ -0,0 +1,225 @@
+// Util classes
```

Comment:
Please, remove filename to 1_repay_loan_eth_4_payments.js (or similar). So we know it uses ETH as collateral.

Reply:
renamed